pkgs/applications/networking/patchwork/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, system, stdenv, fetchurl, nodejs }:
+
+with stdenv.lib;
+
+let
+  nodePackages = import ./composition.nix {
+    inherit pkgs system nodejs;
+  };
+
+  xorg = pkgs.xorg;
+in
+nodePackages.package.override (oldAttrs: {
```

Comment:
I don't think it's necessary to place all of this into its own file, for overriding purposes https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/default-v6.nix can be used.
After that it should be enough to add an alias to `all-packages.nix`.

Reply:
I would love to get it to build first, then we can discuss where it should be placed. :sweat_smile:

Reply:
the package itself should be also added to https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/node-packages-v6.json eventually.